api/python/provisioner/config.py

```diff
@@ -113,6 +113,7 @@
 # FIXME EOS-12334 should be inside factory installation directory
 #    relative paths
 PRVSNR_USER_FILES_SWUPDATE_REPOS_DIR = Path('misc_pkgs/swupdate/repo/files')
```

Comment:
`PRVSNR_USER_FILES_SWUPDATE_REPOS_DIR = Path('misc_pkgs/swupdate/repo/files')`

Should be dropped. Like mentioned earlier, lets not maintain redundant paths.

Reply:
let's postpone for the time we would would have confidence in newer sw upgrade repo mounting. @dmitrydb  could you please create a tack to get rid of older sw update logic to remove the duplication

Reply:
Corresponding ticket for that:
https://jts.seagate.com/browse/EOS-16932